torch_xla/experimental/pjrt.py

```diff
@@ -214,3 +215,20 @@ def run_multiprocess(fn: Callable[..., R], *args,
     }
 
   return results
+
+@requires_pjrt
```

Comment:
Nit: this doesn't _actually_ require `pjrt`. Although this will be the recommended way to synchronize model parameters under PJRT, this is still compatible with XRT